### PR TITLE
Platform.sh CLI image

### PR DESCRIPTION
**Pull request by Mallear**
Docker image with Platform.sh CLI and python3.6.

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-docker_hub_namespace: &docker_hub_namespace ekino/docker-buildbox
+docker_hub_namespace: &docker_hub_namespace mallear/docker-buildbox
```
Comment (mremi) [inline config.yml]: ?

Comment (Mallear) [reply config.yml]: I had to push it to my repo for first use, forgot to revert the change

**Comment by Mallear** [comment]
> Why don't add it to the PHP image?

I need python 3.6 for scripting too

File: config.yml (hunk 1 above)
Comment (Mallear) [reply config.yml]: Change reverted

**Comment by Mallear** [comment]
> What would be the gain of this image when plenty of platformsh-cli exist on dockerhub ?https://hub.docker.com/search?q=platformsh&type=image

Most of them doesn't have python interpreter & when they have, only one "latest" tag is available. This way we control python and tools version & sources.

**Pull request merged**